underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsPositionedUnderFileInputStream.java

```diff
@@ -24,11 +24,21 @@
  * uses the positionedRead {@link FSDataInputStream} API. This stream can be cached for reuse.
  */
 public class HdfsPositionedUnderFileInputStream extends SeekableUnderFileInputStream {
+  private static final int SEQUENTIAL_READ_LIMIT = 3;
+  private static final int MOVEMENT_LIMIT = 512;
```

Comment:
Do these numbers require tuning or is the heuristic good enough?

Reply:
We could further tune these, but experimentally journal replays are as fast as before, will verify remote spark tpcds behavior soon.

Reply:
nit: add a comment describing these as counts and not a bytes limit

Reply:
if they require tuning, i prefer and property. we can choose to make it hidden in case we don't want to add to the user's confusion.

Reply:
So I am actually leaning towards making these not tunable for now. A tunable parameter is only useful if there is some metric / log message that suggests these parameters need to be tuned. Right now we don't have any of that. These default values should work well out of the box. 

How about we add a TODO here to make another PR with the appropriate diagnostic metrics and make it tunable in that PR?